src/main/java/net/coderbot/iris/pipeline/newshader/TriforcePatcher.java

```diff
@@ -193,6 +194,37 @@ public static String patch(String source, ShaderType type, AlphaTest alpha, bool
 		ProgramBuilder.MACRO_CONSTANTS.getDefineStrings().forEach(defineString ->
 				transformations.injectLine(Transformations.InjectionPoint.DEFINES, defineString + "\n"));
 
+		// Inject the legacy lines transformation after everything else
+		if(type == ShaderType.VERTEX && injectLegacyLines) {
+			transformations.injectLine(Transformations.InjectionPoint.BEFORE_CODE, "uniform float iris_LineWidth;");
+			// TODO: Need a better way to do this
+			transformations.replaceExact("uniform vec3 cameraPosition;", ""); // Required if shader has cameraPosition inside a guard
+			transformations.injectLine(Transformations.InjectionPoint.BEFORE_CODE, "uniform vec3 cameraPosition;");
+
+			transformations.replaceExact("uniform float viewWidth;", "");
+			transformations.replaceExact("uniform float viewHeight;", "");
+			transformations.replaceExact("uniform float viewWidth, viewHeight;", ""); // This is sooo scuffed
```

Comment:
It would be better to avoid comments like these, and replace them with more useful info.

Reply:
Is this better now?

Reply:
Yep.